airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshTunnel.java

```diff
@@ -311,7 +312,8 @@ KeyPair getPrivateKeyPair() throws IOException, GeneralSecurityException {
     if (keyPairs != null && keyPairs.iterator().hasNext()) {
       return keyPairs.iterator().next();
     }
-    throw new RuntimeException("Unable to load private key pairs, verify key pairs are properly inputted");
+    throw new ConfigErrorException("Could not connect with provided SSH configuration. Error: "
+        + "Unable to load private key pairs, verify key pairs are properly inputted");
```

Comment:
Is there supposed to be some argument in between the string concatenation or is this supposed to be multiline string? If the latter it may be cleaner to have a text block similar to this for readability https://github.com/airbytehq/airbyte/blob/9c2601013771ca0cd83b633303d40dcbc1a38b1a/airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/operations/RedshiftSqlOperations.java#L72-L77